MDSassignment/guestbook/post/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+from .models import Post
+
+def hide_post(modeladmin, request, queryset):
+	queryset.update(status='h')
+hide_post.short_description = "Hide this post"
+
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ['name', 'status', 'text', 'email', 'time']
+    ordering = ['time']
+    actions = [hide_post]
```

Comment:
I love that you've made this an admin action for marking multiple posts as spam at the same time :+1: